utils.py

```diff
@@ -57,6 +72,16 @@ def save_model(model, epoch):
     model.save_weights(os.path.join(
         config.SAVED_MODELS_PATH, f"{model.name}_{epoch:06d}{dev_mode_string}.h5"))
 
+def load_model(file_path, batch_size=1):
+    type = os.path.split(file_path)[-1].split("_")[0]
+    if type == "BasicCnn":
+        model = BasicCNN()
+    if type == "PianocktailGru":
+        model = PianocktailGRU()
```

Comment:
I think we could use the enum that is defined in `config.py` here. Maybe we could move this enum in a separate file in model folders.
Let's do it in another PR though.